pages.la/common/tldr.md

```diff
@@ -0,0 +1,20 @@
+# tldr
+
+> ສະແດງຕົວຢ່າງແບບງ່າຍສຳລັບເຄື່ອງມືເທິງ command-line ຈາກໂປຣເຈັກ tldr-pages.
+> ຂໍ້ມູນເພີ່ມເຕີມ: <https://tldr.sh>.
+
+- ສະແດງຕົວຢ່າງການໃຊ້ຄຳສັ່ງທີ່ໃຊ້ເລື້ອຍໆ (ຄຳໃບ້: ນີ້ເປັນເຫດຜົນທີ່ທ່ານໃຊ້ບໍລິການຂອງເຮົາແມ່ນບໍ່ລ່ະ!):
```

Comment:
I checked this description with google translate this is what it read "hint: this is why you use our service, isn't it!" shouldn't it be "hint: this is how you got here!"

Can you change this description too?